intcoder.go

```diff
@@ -101,11 +102,16 @@ func (c *chunkedIntCoder) Add(docNum uint64, vals ...uint64) error {
 
 // Close indicates you are done calling Add() this allows the final chunk
 // to be encoded.
-func (c *chunkedIntCoder) Close() {
-	encodingBytes := c.chunkBuf.Bytes()
-	c.chunkLens[c.currChunk] = uint64(len(encodingBytes))
-	c.final = append(c.final, encodingBytes...)
+func (c *chunkedIntCoder) Close() error {
```

Comment:
Same as other coder, Can use use something like `Flush()` instead? i am not sure if calling any method after `Close()` looks correct. @mschoch  for your opinion

Reply:
Like above, I think is ok what you said, I wrote like this just reference to what you wrote like `contentcoder.go`. I think i should keep the code style as before.

Reply:
and this file was wrote by @mschoch , `contentcoder.go` `intcoder.go` and `documentcoder.go` i think we should keep same style.